### DISCUSSION TITLE
Auto Updater

### DISCUSSION THREAD

**Discussion by DanL58** [Courseplay/Courseplay_FS22#642] [Q&A] [answered]
With the FS19 version of Courseplay there was a Auto Update bat file that would look for and install the latest version of Courseplay. Is there any similar updater for FS22 Courseplay yet? Thanks

**Comment by mario-fehr** [comment, marked as answer]
@DanL58 here https://github.com/Courseplay/Courseplay_FS22/discussions/540#discussioncomment-1996191 but I don't know if it is trustable

**Reply by DanL58** [reply]
Thanks but not sure if I should try it. The updater I had for FS19 was from the Courseplay developers and was available here on Github.

**Reply by Tensuko** [reply]
> The updater I had for FS19 was from the Courseplay developers

Thats wrong!
The Auto Update from the FS19 Version was as well made by a User. But it was trustable as he also explained how it works and we also could suggest him changes.

We always suggest you to stick to the Release Version anyway. Wich is just a fast and easy download.



> @DanL58 here [#540 (comment)](https://github.com/Courseplay/Courseplay_FS22/discussions/540#discussioncomment-1996191) but I don't know if it is trustable

That's exactly why I don't promote it on the Main Issue Page :)

**Reply by fudoido** [reply]
I remember there was one for fs17 that I modified a setting and it was usable for fs19 also, maybe it can be changed again for fs22. However, that auto updater was hosted in the now extinct fs-uk website, so I don't know if the one mentioned here now for fs19 is the same.

**Reply by shadovwv** [reply]
Hi, I made this exe file myself which you mentioned earlier, for simple use. Here is a link to that script so you can check if everything is ok and thanks again for such a great mod as Courseplay.(sorry for my english it's not my native language)
https://gist.github.com/shadovwv and the same for the Autodrive. Link for exe files https://github.com/shadovwv
